cherryontech/unmatched
# #UM-26/27 Professionals and Bio pages responsiveness

## Description

This PR modifies components within `BioPage` and `ProfessionalsPage` so that they're responsive for all screen sizes.

## Related Tickets & Documents

[Fixes UM #26](https://lemon-zest.atlassian.net/jira/software/projects/UM/boards/2?selectedIssue=UM-26)
[Fixes UM #27](https://lemon-zest.atlassian.net/jira/software/projects/UM/boards/2?selectedIssue=UM-27)

## Acceptance Criteria



## What type of PR is this? (check all applicable)


|     | Type                       |
| --- | -------------------------- |
|     | :sparkles: New feature     |
|✓| 🎨 Style                   |
|     | :hammer: Refactoring       |
|     | 🔥 Performance Improvements|
|     | :bug: Bug fix              |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |


## Mobile & Desktop Screenshots/Recordings

| Desktop view                                   | Tablet view                                   | Mobile view                                   |
|--------------------------------------- |------------------------------------|-----------------------------------------------------|
| <img src="https://github.com/cherryontech/unmatched/assets/106587342/59bacd9f-40a4-4b92-9b55-9c8e319ee70c" width="250" alt="Desktop view screenshot of the unmatched voices page on the unmatched website"> |<img src="https://github.com/cherryontech/unmatched/assets/106587342/b1b2946d-f700-40ef-9aaf-d536b642b10d" width="200" alt="Tablet view screenshot of the unmatched voices page on the unmatched website">|<img src="https://github.com/cherryontech/unmatched/assets/106587342/e23c7e48-aa9b-4607-8558-0ec653c225ff" width="150" alt="Mobile view screenshot of the unmatched voices page on the unmatched website">|
| <img src="https://github.com/cherryontech/unmatched/assets/106587342/dd6430e8-0f84-491f-97aa-58d9922ab3ca" width="250" alt="Desktop view screenshot of the bio page on the unmatched website"> |<img src="https://github.com/cherryontech/unmatched/assets/106587342/329baec5-a000-4ab8-ad08-60879f57b533" width="200" alt="Tablet view screenshot of the bio page on the unmatched website">|<img src="https://github.com/cherryontech/unmatched/assets/106587342/3a44d1c8-37fa-4e24-81ff-1e0cd6436dbc" width="150" alt="Mobile view screenshot of the bio page on the unmatched website">|

## What I learned

I learned so much more about styling with tailwind. Specifically how to work with the `xl:` `lg:` `md:` `sm:` utility classes.

## Testing steps
1. Install dependencies: `npm i`
2. Run `git checkout fix/responsiveness`
3. Run localhost: `npm run dev`
4. View the app in browser and navigate to `/professionals` page. Test responsiveness for the page using devtools. Press a professionals card and you'll see the "bio page" (e.g., `/professionals/1/Reshma%20Saujani`) with information about them, resources, and other recommended professionals. Test responsiveness again for the page using devtools.


## Added to documentation?

|     | Type                       |
| --- | -------------------------- |
|     | 📜 README.md    |
|     | 🙅 no documentation needed    |


## What gif best describes this PR or how it makes you feel?
![purple oval with spikes that has the word 'yay' on it](https://media.giphy.com/media/1hAxHkVd5divX4fx4z/giphy.gif)
